conformance/results/pyre/classes_classvar.toml

```diff
@@ -3,6 +3,7 @@ notes = """
 Does not reject use of TypeVar in ClassVar.
 Does not reject use of ParamSpec in ClassVar.
 Does not reject use of ClassVar as a generic type argument.
+Does not allow ClassVar to be used without an assigned value.
```

Comment:
I don't think pyre should be penalized as non-compliant for attempting to error on uninitialized ClassVar. It still appears to infer the type as `Any`, which is the point of this change to the spec. Erroring on unitialized attributes is an additional feature, which is not specified but also not against the spec.